iambic/plugins/v0_1_0/aws/organizations/scp/utils.py

```diff
@@ -0,0 +1,572 @@
+from __future__ import annotations
+
+import asyncio
+import json
+import random
+from itertools import chain
+
+from deepdiff import DeepDiff
+from git import TYPE_CHECKING
+
+from iambic.core.context import ctx
+from iambic.core.logger import log
+from iambic.core.models import ProposedChange, ProposedChangeType
+from iambic.core.utils import NoqSemaphore, aio_wrapper, plugin_apply_wrapper
+from iambic.plugins.v0_1_0.aws.utils import boto_crud_call, legacy_paginated_search
+
+if TYPE_CHECKING:
+    from iambic.plugins.v0_1_0.aws.models import AWSAccount
+    from iambic.plugins.v0_1_0.aws.organizations.scp.models import (
+        ServiceControlPolicyItem,
+        ServiceControlPolicyTargetItem,
+    )
+
+
+async def list_policies(
+    client, filter="SERVICE_CONTROL_POLICY"
+) -> list[ServiceControlPolicyItem]:
+    """Retrieves the list of all policies in an organization of a specified type."""
+
+    from iambic.plugins.v0_1_0.aws.organizations.scp.models import (
+        ServiceControlPolicyItem,
+    )
+
+    scp_policies = await legacy_paginated_search(
+        client.list_policies,
+        response_key="Policies",
+        **dict(Filter=filter),
+    )
+
+    scp_policies = [p for p in scp_policies if p["AwsManaged"] is False]
+
+    list_targets_for_policy_semaphore = NoqSemaphore(list_targets_for_policy, 10)
+    describe_policy_semaphore = NoqSemaphore(get_policy_statements, 10)
+    list_tags_by_policy_semaphore = NoqSemaphore(list_tags_by_policy, 10)
+
+    # CHECK: come in order by scp_policies ?
```

Comment:
Is the comment needed? If so, I think it needs to be clearer so others can understand

Reply:
just an old comment